src/components/search/screens/SearchInput/views/SearchInputView.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Header, Item, Icon, Input, Button, Text } from 'native-base';
+import { GenericHeader } from 'yasav/src/viewElements/shared/Header';
+import I18n from 'yasav/locales/i18n';
+
+export default class SearchInputView extends React.Component {
+  constructor(props) {
+    super(props);
+    this.doSearch = this.doSearch.bind(this);
+  }
+  
+  doSearch(request) {
+    this.props.doSearch(request)
+  }
+  
+  render() {
+    return (
+      <Header searchBar rounded>
+        <Item>
+          <Icon name="ios-search" />
```

Comment:
Following the doc [here](https://ionicframework.com/docs/ionicons/), just putting `name="search"` will select the iOS icon on iPhones and the Material Design icon on Android, which is better !